### DISCUSSION TITLE
Can enter setup on windows 11

### DISCUSSION THREAD

**Discussion by Rumarofi** [vhanla/winxcorners#42] [General]
Hi. Maybe I'm doing something wrong but the icon on the taskbar never shows to me.

I've run the executable and I can find it on the process list. however, I can't do anything else because the icon that should show up on the right side of the task bar, it's not there.

I've tried killing the process and run it again but the result is the same.

Any idea of what may be wrong?

Thank you in advance.

**Comment by peterinjapan** [comment]
I got it to work in Windows 11. I finally found the executable, thanks for pointing me towards it. After clicking it, it's there in the system tray and you can click it to set the four corner actions.
